core/senders/qq_file_fallback.py

```
@@ -0,0 +1,224 @@
+        return False
+
+    original_name = _get_file_component_name(component)
+
+    if policy.mode in {"link", "link_or_zip"} and policy.direct_link_base_url:
+        direct_link = _build_direct_link(policy.direct_link_base_url, original_name)
+        await send_message_fn(
+            unified_msg_origin,
+            MessageChain(
+                [
+                    Plain(
+                        f"原文件 {original_name} 因 QQ 文件限制无法直传，改发直链：{direct_link}"
```

Ruff (0.15.12)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)